main/lsp/LSPTypechecker.cc

```diff
@@ -143,6 +221,7 @@ TypecheckRun LSPTypechecker::runFastPath(LSPFileUpdates updates, WorkerPool &wor
         unique_ptr<KeyValueStore> kvstore; // nullptr
         // TODO: Thread through kvstore.
         ENFORCE(this->kvstore == nullptr);
+        // TODO(jvilk): We don't need to re-index files that didn't change.
```

Comment:
I noticed that we're doing extra work on the fast path right now, so I added a little TODO.

This may be a noticeable perf issue if you change the sig of a method called in many files.